### ISSUE TITLE
Could we improve how these codeblocks of file structures are read to screenreaders?

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#186443]
Testing #186218

Query:
```
/createWorkspace create a PowerShell module product that wraps the Open Exchange Rates API
```

Result:
<img width="1563" alt="image" src="https://github.com/microsoft/vscode/assets/2644648/6bdf68bc-629b-4490-adfd-450d358b446e">

To help sighted users, we response with this tree-like structure which makes a lot of sense. Unfortunately, for someone who cannot see this, it's very confusing where folders start and where files end.

Could these blocks be read out differently?

Could we have chat responses come back in a screen reader friendly way?

**Comment by meganrogge** [comment]
There's an `Open Accessible View` command which allows screen reader users to inspect line by line, char by char. it could use some refinement though

**Comment by meganrogge** [comment]
@TylerLeonhardt can you try `alt+f2` and see if it addresses your concern?

**Comment by TylerLeonhardt** [comment]
![image](https://github.com/microsoft/vscode/assets/2644648/9c7e17f3-3577-4b6f-9d42-f646570c1855)
here's what that looks like. It doesn't read any different. Was there something in particular you wanted me to try?

**Comment by meganrogge** [comment]
Yes that's different because it allows them to read it line by line instead of as one massive and overwhelming chunk. 

We could think abt parsing out the lines in that tree, but not sure those get read by the screen reader anyway. I'll have to check

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by meganrogge** [comment]
loosely related https://github.com/microsoft/vscode/issues/188702

**Comment by meganrogge** [comment]
Thanks to @joyceerhl, this is now very accessible

<img width="420" alt="Screenshot 2023-10-23 at 9 21 02 AM" src="https://github.com/microsoft/vscode/assets/29464607/a5ff3de4-cb42-483c-8600-511cd240cc63">
